site/content/en/guides/consumer/function/export/tekton/_index.md

```diff
@@ -8,9 +8,15 @@ description: >
 
 In this tutorial, you will pull an example blueprint that declares Kubernetes resources and two kpt functions. Then you will export a pipeline that runs the functions against the resources on [Tekton](https://tekton.dev/) and modify it to make it fully functional. How to setting up Tekton is also included if you don't have one running yet. This tutorial takes about 20 minutes.
 
+{{% pageinfo color="info" %}}
+A kpt version `v0.32.0` or higher is required.
+{{% /pageinfo %}}
+
 ## Before you begin
 
```

Comment:
Explicitly call out that you need `tkn` installed before you begin and link to the release page.